shared/datastore.go

```diff
@@ -181,24 +218,45 @@ func contains(s []string, x string) bool {
 	return false
 }
 
-// Loads any keys for a full string match or a version prefix (Between [version].* and [version].9*)
-func loadKeysForBrowserVersion(ctx context.Context, query *datastore.Query, version string) (result *mapset.Set, err error) {
+// Loads any keys for a revision prefix or full string match
+func loadKeysForRevision(ctx context.Context, query *datastore.Query, sha string) (result []*datastore.Key, err error) {
```

Comment:
Looks like all the call sites convert the return value to `IntID`. Shall we just return the int ID (also for consistency with the other `load*` function)?

Reply:
Done.